src/hooks/use-authentication/index.ts

```diff
@@ -83,6 +84,27 @@ const useAuthentication = (
 		session = { ...data }
 		user = data.user
 		delete session.user
+
+		const segmentUserId = safeGetSegmentId()
+
+		if (canAnalyzeUser() && segmentUserId !== session.id) {
+			analytics?.identify(
+				session.id,
+				{
+					email: user.email,
+					name: user.name,
+					devPortalSignUp: true,
+					createdAt: Date.now().toString(),
+				},
+				{
+					// This limits PII flow to only Marketo as per Security and data engineering
+					integrations: {
+						All: false,
+						'Marketo V2': true,
+					},
+				}
```

Comment:
Let's remove this so we can test the anonymous ID passing and see how it works with identified users.